src/api/routes/cards.js

```diff
@@ -188,4 +202,12 @@ router.delete('/:cardId/labels/:labelId', (req, res) => {
   })
 })
 
+router.delete('/:cardId/responsible/', (req, res) => {
```

Comment:
You need to be coherent this the field updated, choose between `cardResponsible` or `responsible`. I prefer `responsible` because we already know that we are in the context of a card.